duckduckgo/iOS
# filter out networks with less than one percent from the leaderboard

Task/Issue URL: https://app.asana.com/0/414709148257752/1139123263078640
Tech Design URL:
CC:

**Description**:

Only show networks seen on >= 1% of pages visited.

**Steps to test this PR**:
1. Visit a bunch of sites
1. View network leaderboard
1. Ensure none show "0%"


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
